packages/babel-helpers/src/helpers.js

```diff
@@ -377,7 +377,7 @@ helpers.objectSpread = () => template.program.ast`
 
   export default function _objectSpread(target) {
     for (var i = 1; i < arguments.length; i++) {
-      var source = (arguments[i] != null) ? arguments[i] : {};
+      var source = arguments[i] != null && arguments[i] !== false ? arguments[i] : {};
```

Comment:
We can fix this and speed up all non-objets:

```js
var source = arguments[i];
if (source !== Object(source)) continue;

var ownKeys = Object.keys(source);
//...

Reply:
I believe that strings should be spread.
The check could be `if (!source) continue;`

Reply:
If we do want to do this, which I'm not convinced we do, it seems like we should approach it as including our own polyfill for `Object.keys`, but really it seems not too hard to tell people to load their own polyfills?

Reply:
I discovered the problem in the Bootstrap 4 framework that uses babel.